src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig

```diff
@@ -33,17 +33,18 @@
             {% endblock %}
 
             {% set currentRoute = app.request.attributes.get('_route') %}
+            {% set currentRouteFromQueryParams = app.request.query.get('currentRoute') %}
```

Comment:
Can we a centralized place where we defined which route is active? It might be less messy in the html.
Like:

```twig
{% if currentRoute == 'unread' or currentRoute == 'homepage' or currentRouteFromQueryParams == 'unread' %}
    {% set activeRoute = 'unread' %}
{% elseif currentRoute == 'starred' or currentRouteFromQueryParams == 'starred' %}
    {% set activeRoute = 'starred' %}
{% endif %}
```